tests/test_tox_to_nox.py

```diff
@@ -383,7 +382,7 @@ def test_commands_with_requirements(makeconfig: Callable[[str], str]) -> None:
 
     @nox.session(python='python3.13')
     def aiohttp(session):
-        session.install('pytest', 'pytest-cov', '-r', 'requirements/aiohttp.txt')
+        session.install('pytest', 'pytest-cov', '-rrequirements/aiohttp.txt')
```

Comment:
I guess this is the reason of the failing checks :)

```suggestion
        session.install('pytest', 'pytest-cov', '-r', 'requirements/aiohttp.txt')
```

Reply:
The problem is that the older tox is now repairing this as one joined string. I think we just need to make the tests work in either case, as they are identical.